.github/workflows/linters.yml

```diff
@@ -31,4 +31,35 @@ jobs:
         with:
           options: "--check --verbose --line-length 120 --diff --color --target-version py312"
           src: ${{ github.workspace }}/deploy
-          version: "~= 24.0"
\ No newline at end of file
+          version: "~= 24.0"
+  # run_pytest:
+  #   runs-on: ubuntu-latest 
+  #   steps:
+  #     - name: Checkout code
+  #       uses: actions/checkout@v4
+  #     - name: Set up Python
+  #       uses: actions/setup-python@v4
+  #       with:
+  #         python-version: '3.12.2'
```

Comment:
please use `3.12.8` which is a actual version of 3.12 track, or just define `3.12` to use the latest 3.12.x

ref: https://www.python.org/downloads/release/python-3128/